ts/components/ListSelectionBar.tsx

```diff
@@ -62,14 +58,14 @@ export class ListSelectionBar extends React.PureComponent<Props> {
             bordered={true}
             light={true}
             onPress={onResetSelection}
-            style={styles.buttonBarLeft}
+            style={styles.flex2}
```

Comment:
Could you check if this component still needs to be wrapped inside a **_Animated.View_** component?